### PR TITLE
Added closet + improved AI

### LINKED ISSUE #18: Implement Hiding

### PR DESCRIPTION
**Pull request by tulza**
closes #18

### CODE REVIEW

File: Scenes/Objects/closet.gd [hunk 1]
```diff
@@ -0,0 +1,52 @@
+extends StaticBody2D
+class_name Closet
+
+var player: Player
+var is_player_hiding:bool = false
```
Comment (AntGa) [inline Scenes/Objects/closet.gd:5]: inconsistent spacing 🤓

**Review by AntGa — APPROVED** [review]
lgtm i pray for your merge conflicts

File: Scenes/Objects/closet.gd (hunk 1 above)
Comment (BlackPri0r) [reply Scenes/Objects/closet.gd:5]: Fixed. LGTM!

Comment (AntGa) [reply Scenes/Objects/closet.gd:5]: you're not even the one pull requesting 😭

**Pull request merged**